### DISCUSSION TITLE
Neo4j backup script Issue

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#535] [Bugs]
HI team!

We are trying to take neo4j backup using OpsAdministration/test/KnowledgePlatform/Neo4jBackup.  job got success, but while extracting the backup tar we are getting an empty folder.
It seems like backup startup file inside the bin folder is not there.

REF: https://github.com/Sunbird-Knowlg/sunbird-learning-platform/blob/release-4.8.0/ansible/roles/neo4j-backup/templates/backup_script.j2#L3

Could you please suggest.

Regards,
Gohila

**Comment by chitranshu-keshav** [comment]
@santhosh-tg @tsprasath Can you please get back on the above issue

**Reply by Krishnaj20** [reply]
@gandham-santhosh - for your attention please.

**Reply by santhosh-tg** [reply]
@gohilamariappan neo4j-backup script is part of enterprise edition and not available in the community one.  Please refer the **neo4j-admin** command for options to take backup and restore.

**Comment by gohilamariappan** [comment]
Hi @santhosh-tg 

I have tried neo4j-admin in the same script.  its not working. below one is the error
####################
unrecognized command: --from=127.0.0.1:7362
###########

Please suggest.

**Reply by santhosh-tg** [reply]
Like i mentioned the backup script is not for community edition. You should try running the command manually to take the backup and not just replacing the command in the script.

**Reply by Krishnaj20** [reply]
@santhosh-tg - Please check if the query is answered and take it to closure.
